damus/Views/NoteContentView.swift

```diff
@@ -181,6 +243,112 @@ func mention_str(_ m: Mention, profiles: Profiles) -> String {
 }
 
 
+public struct Translator {
```

Comment:
Sorry if this is gross. I may refactor this later for translating other user-authored things in the future. Like bios in people's profiles, DMs, etc.